shoov/modules/custom/shoov_ci_build/shoov_ci_build.module

```diff
@@ -131,6 +131,24 @@ function shoov_ci_build_node_presave($node) {
 
   $wrapper = entity_metadata_wrapper('node', $node);
 
+  if ($node->is_new) {
+    // Check config file exists for the new CI Build.
+    if (!shoov_repository_config_file_exists($wrapper->og_repo->value(), $wrapper->field_git_branch->value())) {
+      throw new Exception('no_config_file');
```

Comment:
exception should be human readble. `".shoov.yml if missing in the root of the repository."`